src/os/interaction/modifyinteraction.js

```diff
@@ -247,62 +482,54 @@ class Modify extends OLModify {
   }
 }
 
+osImplements(Modify, I3DSupport.ID);
+
 
 /**
- * Style for the feature being modified.
- * @type {Array<Style>}
- * @const
+ * The original OL mouse down event handler.
+ * @type {function(MapBrowserPointerEvent):boolean}
+ * @suppress {accessControls} Access to the original OL function.
  */
-Modify.STYLE = [
-  new Style({
-    stroke: new Stroke({
-      color: [0, 153, 255, 1],
-      width: 3
-    }),
-    image: new Circle({
-      radius: 6,
-      fill: new Fill({
-        color: [0, 153, 255, 1]
-      }),
-      stroke: new Stroke({
-        color: [255, 255, 255, 1],
-        width: 2
-      })
-    }),
-    zIndex: Infinity
-  })
-];
+const oldHandleDownEvent = OLModify.handleDownEvent_;
 
 
 /**
- * Style for the vertex feature.
- * @type {Array<Style>}
- * @const
+ * Mixin to the OL mouse down event handler.
+ * @param {MapBrowserPointerEvent} evt Event.
+ * @return {boolean} Start drag sequence?
+ * @this {Modify}
+ * @private
+ *
+ * @suppress {accessControls} Replace OL function and access vertex feature.
  */
-const VERTEX_STYLE = [
-  new Style({
-    image: new Circle({
-      radius: 6,
-      fill: new Fill({
-        color: [0, 153, 255, 1]
-      }),
-      stroke: new Stroke({
-        color: [255, 255, 255, 1],
-        width: 2
-      })
-    }),
-    zIndex: Infinity
-  })
-];
+OLModify.handleDownEvent_ = function(evt) {
```

Comment:
Will this kind of mixin/override work once we take `declareLegacyNamespace` out of this file? We can worry about that when we get to it, just might add a note if it's a concern.

Reply:
Yeah it will work until we update OpenLayers, which will probably break all of our mixins. All `declareLegacyNamespace` is doing is adding the module's exports to `window` using the module's namespace, so legacy `goog.provide` files can reference it. Removing it won't change any interaction with `goog.provide` files from OL/Closure/etc.